masonry/src/widgets/grid.rs

```diff
@@ -52,25 +56,23 @@ impl Grid {
         }
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Builder-style method for setting the spacing between grid items.
     pub fn with_spacing(mut self, spacing: f64) -> Self {
         self.grid_spacing = spacing;
         self
     }
 
-    /// Builder-style variant of [`Grid::add_child`].
```

Comment:
I do think it's useful to link to the runtime equivalent; see how I've documented things on `TextArea` for example.
That doesn't need to be in this PR though; it's a follow-up cleanup pass. And I agree that the old wording here was actively misleading, so changing it is good.